android/build.gradle

```diff
@@ -0,0 +1,34 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:1.1.3'
+  }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+  compileSdkVersion 23
+  buildToolsVersion "23.0.1"
+
+  defaultConfig {
+    minSdkVersion 16
+    targetSdkVersion 22
```

Comment:
I would bump `minSdkVersion` and `targetSdkVersion` up. FingerPrint works only on Android 23 and above, right? So it does not make really sense we say `16`.